discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+    async def removeadminrole(self, interaction: Interaction, role: Role):
+        if interaction.guild:
+            if not role in interaction.guild.roles:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Could not find role: {role.name}",
+                        colour=Colour.blue(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            session: SQLAlchemySession
+            with Session() as session:
+                admin_role = (
+                    session.query(AdminRole)
+                    .filter(AdminRole.role_id == role.id)
+                    .first()
+                )
+                if admin_role:
+                    session.delete(admin_role)
+                    session.commit()
+                    await interaction.response.send_message(
+                        embed=Embed(
+                            description=f"Removed admin role: {role.name}",
+                            colour=Colour.green(),
+                        )
+                    )
+                else:
+                    await interaction.response.send_message(
+                        embed=Embed(
+                            description=f"Could not find admin role: {role.name}",
+                            colour=Colour.blue(),
+                        ),
+                        ephemeral=True,
+                    )
+
+    @group.command(name="removecommand", description="Remove a custom command")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(name="Name of existing custom command")
+    async def removecommand(self, interaction: Interaction, name: str):
+        session: SQLAlchemySession
+        with Session() as session:
+            exists = (
+                session.query(CustomCommand).filter(CustomCommand.name == name).first()
+            )
+            if not exists:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Could not find command with that name",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            session.delete(exists)
+            session.commit()
+
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Command `{name}` removed",
+                    colour=Colour.green(),
+                )
+            )
+
+    @group.command(name="removedbbackup", description="Remove a database backup")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(db_filename="Name of backup file")
+    async def removedbbackup(self, interaction: Interaction, db_filename: str):
```

Comment:
the db specific commands don't work with postgres anymore, so it should be marked clearly both with a comment and an early return inside of the function that tells the user so.

Reply:
Added check on `config.DATABASE_URI` to return early if setup for postgres.
Added comments & `TODO`.

Similarly updated `createdbbackup()`